clientlibs/js/src/functions/init.ts

```diff
@@ -9,32 +10,36 @@ export default async function init(
   group?: Record<string, Array<string>>,
   workingGroup?: Record<string, string>
 ): Promise<Interfaces.IUser> {
-  try {
-    let data: any = {
-      id: userId,
-    };
+  let data: any = {
+    id: userId,
+  };
 
-    if (group) {
-      data = {
-        ...data,
-        group,
-      };
+  if (group) {
+    if (!(group instanceof Map)) {
```

Comment:
Now we are taking group and working group as an object instead of Map so this part of the code is incorrect. Ideally it should throw an error if you are passing object for group and working group. Remove the checking of instance of Map